atom/common/api/resources/windows_bindings.js

```diff
@@ -21,7 +21,16 @@ binding.registerCustomHook(function (bindingsAPI, extensionId) {
   })
 
   apiFunctions.setHandleRequest('create', function (createData, cb) {
-    console.warn('chrome.windows.create is not supported yet')
+    var responseId = ++id
```

Comment:
we should be using `ipc.guid()` here. The integer ids can be duplicated across windows. Please update other uses in this file as well.

Reply:
if you have time please also update browser_action_bindings.js in a separate PR

Reply:
looks like also context_menu_bindings.js

Reply:
addressed 3a9b420023fa87de8299273b3023053a9adc8159

Reply:
sure will do